load.php

```diff
@@ -162,9 +162,36 @@ function( $module_settings ) {
 	 */
 	$modules = apply_filters( 'perflab_active_modules', $modules );
 
+	if ( ! function_exists( 'is_plugin_active' ) ) {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+	}
+
+	$standalone_plugin_config = perflab_get_standalone_plugins_config();
+	foreach ( $standalone_plugin_config as $plugin_slug => $plugin_main_file ) {
+		if ( in_array( $plugin_slug, $modules, true ) && is_plugin_active( $plugin_main_file ) ) {
```

Comment:
The `is_plugin_active()` function is not available in frontend requests, so we shouldn't use it. I see you load it above, but that's not a good practice, loading admin functions in every frontend request. We have to find another solution.

I wonder, maybe the most reliable is to introduce a constant at the top of each module's `load.php` file (e.g. `WEBP_UPLOADS_VERSION`, `SQLITE_INTEGRATION_VERSION` etc.) and then we could check whether the constant is already defined in this logic.

While it involves adding the constants to the modules / plugins, I think this will be the most reliable approach. Sometimes plugins can be loaded in another folder than their expected slug, in which case the constant approach will still work, but relying on the plugin slug won't.

Reply:
@felixarntz I try the approach you suggested for constant but i fall in same error when i try to active PL plugin after the WebP Uploads plugin.

Even if i define constant at top of the file i get same duplicate function error. Please guide me if i'm on wrong track or did not get your feedback

Reply:
@mukeshpanchal27 Sorry, I meant the constant should be defined in a way that they don't conflict:
```php
if ( ! defined( 'WEBP_UPLOADS_VERSION' ) ) {
    defined( 'WEBP_UPLOADS_VERSION', 'the.version.number' );
}
```

Reply:
Thanks @felixarntz, I have implemented constant approach as per suggestion in  https://github.com/WordPress/performance/pull/670/commits/f93cf0f9ca5edeffcca3b6bf408bde4e49d45b56.

When i first activate `WebP Uploads` plugin then `Performance Lab`, It still get defined value for PL plugin instead of the `WebP Uploads`. So WP load `Performance Lab` plugin first then `WebP Uploads`.

Reply:
@mukeshpanchal27 Ahh right, I forgot about that. We need to change the function call to load the modules to not happen immediately in `load.php`, but on the `plugins_loaded` action instead.

Reply:
Thanks @felixarntz, I have implemented it in https://github.com/WordPress/performance/pull/670/commits/803a26afb68e61ce7d2b30637cdfd9174bc8dd31 and it is working fine for frontend requests.